rpcs/src/methods/chain_head.rs

```diff
@@ -872,6 +885,28 @@ pub enum ArchiveCallResult {
     Error(String),
 }
 
+impl ArchiveCallResult {
+    /// Unwrap a successful value from the call, panicking if it was an error.
+    pub fn unwrap_success(self) -> Bytes {
```

Comment:
I would prefer `fn as_success(self) -> Option<Bytes>` and let the user to the unwrapping themselves.

Anyway, either is fine

Reply:
Good idea; I'll change the APIs to that!